browser/css/jsdialogs.css

```diff
@@ -14,9 +14,31 @@
 	display: flex;
 }
 
-.jsdialog #writerchanges .ui-treeview-body {
-	min-height: 300px;
-	max-height: none;
+#AcceptRejectChangesDialog .ui-grid {
```

Comment:
could you move all these rules to the end of file
and before add separation comment with name of the dialog
that would make things grouped, here we have general rules for all dialogs